code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -9,6 +9,8 @@
 		TRAIT_CAN_STRIP,
 		TRAIT_PLANT_SAFE,
 	)
```

Comment:
You lack mutant_bodyparts here to mark it being a preference for them

Reply:
its an external organ